docs/manual/manual_redundant_throttle.rst

```diff
@@ -38,10 +43,16 @@ There are two types of configurations available to redundant throttle. The first
 source if the active source (the source whose throttles are being applied to module control) goes offline. The second set of parameters defines the priorities of each supported throttle source. You can find the specific parameters 
 at :ref:`throttle_source_manager`. You can access these parameters either using a :ref:`Vertiq API <getting_started_with_apis>` or through the :ref:`IQ Control Center <control_center_start_guide>`. 
 
-The time that a module waits (throttle source timeout) can be configured through the Control Center's tuning tab:
+The time that a module waits, its throttle switchover time, can be configured through the Control Center's tuning tab as ``Throttle Source Timeout``:
 
 .. image:: ../_static/manual_images/redundant_throttle/throttle_source_to.png
 
+.. note:: 
+
+    Please note that if your throttle source switchover time is greater than propeller motor controller's :ref:`timeout <manual_timeout>`, it is possible to experience 
+    unexpected behaviors. In this scenario, it is possible for the module to reach the propeller motor control timeout before the throttle source manager has 

```

Comment:
I wouldn't call them unexpected behaviors, I expect that to happen. Maybe...generally undesirable behaviors or something? Not sure, but it's what I expect, just probably not what people want. I just don't want it to seem like it's an unexpected bug, when really its just the expected interaction of these two things